vividus-plugin-web-app-playwright/src/test/java/org/vividus/ui/web/playwright/action/CookieHelperTests.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2019-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.ui.web.playwright.action;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import com.microsoft.playwright.BrowserContext;
+import com.microsoft.playwright.options.Cookie;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.vividus.ui.web.playwright.BrowserContextProvider;
+import org.vividus.ui.web.playwright.helper.CookieHelper;
+
+@ExtendWith(MockitoExtension.class)
+public class CookieHelperTests
+{
+    private static final String PATH = "/";
+    private static final String ZERO = "0";
+    private static final String COOKIE_NAME = "name";
+    private static final String COOKIE_VALUE = "cookieValue";
+    private static final String DOMAIN = "https://www.domain.com";
+
+    @InjectMocks private CookieHelper cookieHelper;
```

Comment:
it should go after all fields of type `@Mock`

Reply:
done, thanks